example-runner/src/main/java/uk/gov/gchq/palisade/example/runner/config/AkkaClientWrapper.java

```diff
@@ -68,22 +68,14 @@ public AkkaClientWrapper(final AkkaClient client, final Serialiser<T> serialiser
     }
 
     private static String fileNameToResourceId(final String fileName) {
-        File file;
-        if (!Path.of(fileName).isAbsolute()) {
-            try {
-                file = new File(fileName).getCanonicalFile();
-            } catch (IOException ex) {
-                LOGGER.warn("Failed to get CanonicalFile for '{}', using AbsoluteFile instead", fileName, ex);
-                file = new File(fileName).getAbsoluteFile();
-            }
-        } else {
-            file = new File(fileName);
+        try {
+            return AbstractResourceBuilder.create(fileName).toString();
+        } catch (IllegalArgumentException ex) {
+            String userDir = System.getProperty("user.dir") + "/";
+            LOGGER.debug("Caught exception while creating URI", ex);
+            LOGGER.debug("Suspect {} is a relative-path so trying with user.dir '{}' property", fileName, userDir);
+            return Path.of(userDir + fileName)
+                    .toUri().normalize().toString();
         }
-
-        String resourceId = file.toURI().toString();
-        if ((fileName.endsWith("/") || fileName.endsWith("\\")) && !resourceId.endsWith("/")) {
-            resourceId += "/";
-        }
-        return resourceId;
     }
```

Comment:
Does the `fileName` start with `file:` ?
If using the builder, do we need the `catch` and user-dir stuff?
The `.toString(...)` method returns `"AbstractResource [...]"` stuff, i think you mean `.getId()`?
Ideally, this might boil down to something like:
```
// change yaml data to include "file:" at the start of the fileNames
return AbstractResourceBuilder.create(uri).getId()
// or, less nice, but no yaml changes
return AbstractResourceBuilder.create("file:" + fileName).getId()
```